packages/snowpack/src/commands/build.ts

```diff
@@ -249,15 +249,16 @@ export async function command(commandOptions: CommandOptions) {
 
       // When dealing with an absolute import path, we need to honor the baseUrl
       if (isAbsoluteUrlPath) {
-        return path
-          .relative(
-            path.dirname(outLoc),
-            path.resolve(buildDirectoryLoc, removeLeadingSlash(resolvedImportUrl)),
-          )
-          .replace(/\\/g, '/'); // replace Windows backslashes at the end, after resolution
+        resolvedImportUrl = path.relative(
+          path.dirname(outLoc),
+          path.resolve(buildDirectoryLoc, removeLeadingSlash(resolvedImportUrl)),
+        );
       }
 
-      return resolvedImportUrl;
+      if (!resolvedImportUrl.startsWith('.'))
+        resolvedImportUrl = './' + removeLeadingSlash(resolvedImportUrl);
```

Comment:
main fix: need to just add a preceding `./` (we do in other code paths, but not here)